my-app/src/index.js

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { Events } from './pages/events';
+import { Homepage } from './pages/homepage';
+import { Pdp } from './pages/pdp'
 // import { MainPage } from './pages';
 
-ReactDOM.render(
-    <div>Hello World</div>,
-    document.querySelector('#root')
+function AppRouter(){
```

Comment:
remove this function to separate component App, and then use this component here.